### ISSUE TITLE
CodeBERT Pre-trained models trained from scratch.

### ISSUE THREAD

**Issue by donhk1011** [microsoft/CodeBERT#171]
Hi! 
Could you please let me know where I can find the pre-trained models from scratch (random initialization)?
Specifically, I am looking for the checkpoints of PT W/ CODE ONLY (INIT=S) and CODEBERT (MLM, INIT=S) in Table 2?

Thanks!

**Comment by guoday** [comment]
Sorry, we don't save this model.

**Comment by donhk1011** [comment]
Thanks for the reply! Is it possible to get a training code for this?

**Comment by guoday** [comment]
https://github.com/huggingface/transformers/tree/main/examples/pytorch/language-modeling#robertabertdistilbert-and-masked-language-modeling

**Comment by donhk1011** [comment]
Thanks a lot for the pointer!
